build_linux_arm64.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -eu
```

Comment:
You can also do `#!/bin/sh -eu` and drop the set. Not a big difference, but it's an option.